MainTel/MainTel-Non-standard-Operations/MTM2-Non-standard-Procedures/Power-cycle-MTM2-cabinet.rst

```diff
@@ -0,0 +1,174 @@
+
+- Consult with the hardware or system engineers to obtain approval before proceeding with this procedure.
+
+- Ensure you are connected to the control room network as VPN cannot reach it.
+
+- You should only do this procedure if:
+
+  - Following MTM2 Recovery procedure doesn't fix the issue, this means re-starting the control system fails to clear alarms and you are unable to set MTM2 to close-loop.
+
+.. add link to MTM2 Recovery 
+
+  - You cannot ping or ssh the M2 controller  m2-crio-controller01.cp.lsst.org (on TMA) **m2-crio-controller02.cp.lsst.org** (on level 3).
+
+  - You see errors codes 6051 and 6088, or 6051 and 6052, and in some cases error 6056.
+
+- Know the location of the MTM2 cabinet according its installation, which is either in TMA or on level 3.
+
+.. _Post-Condition:
+
+Post-Condition
+==============
+
+.. This section should provide a simple overview of conditions or results after executing the procedure; for example, state of equipment or resulting data products.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Please provide screenshots of the software status or relevant display windows to confirm.
+.. Do not include actions in this section. Any action by the user should be included in the end of the Procedure section below. For example: Do not include "Verify the telescope azimuth is 0 degrees with the appropriate command." Instead, include this statement as the final step of the procedure, and include "Telescope is at 0 degrees." in the Post-condition section.
+
+- MTMM2 GUI displays no alarms, and MTM2 can transition to a close-loop status.
+
+
+.. _Procedure-Steps:
+
+Procedure Steps
+===============
+
+.. This section should include the procedure. There is no strict formatting or structure required for procedures. It is left to the authors to decide which format and structure is most relevant.
+.. In the case of more complicated procedures, more sophisticated methodologies may be appropriate, such as multiple section headings or a list of linked procedures to be performed in the specified order.
+.. For highly complicated procedures, consider breaking them into separate procedure. Some options are a high-level procedure with links, separating into smaller procedures or utilizing the reST ``include`` directive <https://docutils.sourceforge.io/docs/ref/rst/directives.html#include>.
+
+Depending on the location of MTM2 the corresponding steps are:
+
+.. _If-MTM2-is-on-the-TMA:
+
+If MTM2 is on the TMA
+-----------------------
+
+
+.. .. _`tea-pdu`: https://tea-pdu01.cp.lsst.org/ this link currently (May 30, 2024) get a "This site can't be reached"
+
+#. Enter **tea-pdu**, this is the internet interface of the general power distribution (PDU). 
+
+    - You can see a screenshot in :ref:`pdu-m2` figure below.
+
+#. Find the credentials in the **Operators Vault**, under the name PDU Utilities Cabinet. *Note*: The link to tea-pdu address is in this item.
```

Comment:
Emphaiss in PDU Utilities Cabinet with a single asterisk that will be represented as italics.